src/set_theory/game/pgame.lean

```diff
@@ -6,6 +6,7 @@ Authors: Reid Barton, Mario Carneiro, Isabel Longbottom, Scott Morrison
 import data.fin.basic
 import data.list.basic
 import logic.relation
+import logic.small
```

Comment:
You'll need to merge master here and fix imports. This file has been split in the meantime.

Reply:
If you ping me afterwards I'll try to come back and merge. Mostly looks okay otherwise.